lib/features/modeling/behavior/LabelBehavior.js

```diff
@@ -212,7 +212,7 @@ export default function LabelBehavior(
     var label = event.context.connection.label,
         labelAdjustment;
 
-    if (!label) {
+    if (!label || !label.parent) {
```

Comment:
I find an early return here is to be the best solution. The reason for it is that the only case where `connection.updateWaypoints` is executed for a flow having a not-yet-created label is during the copy-paste operation. Since the label will be created after this block, as a part of `elements.create` exactly in the right position, we simply don't need to move it here.

Reply:
Makes sense. Could you add an additional comment regarding this that makes clear which scenarios we're covering here? That helps us to understand what is going on here and why this case is excluded next time we look into the issue.

Reply:
The problem occurs when LayoutConnectionBehavior of GridSnapping module intervenes during elements.create in order to snap waypoints of SequenceFlows. This only happens for SequenceFlows having waypoints.length > 2. This behavior triggers updateWaypoints which is later catched by LabelBehavior in order to correct the label position which is attached to a SequenceFlow. However since the label is not yet created here, since this all happens during elements.create, the operation fails.